suite/tests/CMakeLists.txt

```diff
@@ -4199,9 +4199,22 @@ if (BUILD_CLIENTS)
       "-sb_time_zero" "18444502336406842448"
       "-sb_sysroot"
       "${PROJECT_SOURCE_DIR}/clients/drcachesim/drpt2trace/test_simple.raw")
-    torunonly_api(tool.drpt2trace drpt2trace
+    torunonly_api(tool.drpt2trace.sideband drpt2trace
       "../../clients/drcachesim/drpt2trace/test_simple.expect"
-      "" "${drpt2trace_args}" ON OFF)
+      "" "${drpt2trace_sideband_args}" ON OFF)
+    set(drpt2trace_elf_args "-print_trace"
+      "-raw_pt"
+      "${PROJECT_SOURCE_DIR}/clients/drcachesim/drpt2trace/test_simple.raw/pt.bin"
+      "-pt_cpu_family" "6"
+      "-pt_cpu_model" "85"
+      "-pt_cpu_stepping" "4"
+      "-pt_mtc_freq" "3" "-pt_nom_freq" "37"
```

Comment:
I see there are some args that are common between the elf and sideband case. How about we define a drpt2trace_common_args instead of duplicating the list?

Reply:
Done.